### PR TITLE
TRITON-2411 Move rabbitmq to triton-origin-21.4.0

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,31 +6,35 @@
...
 BUILDIMAGE_PKGSRC = \
-	perl-5.14.2nb3 \
-	iodbc-3.52.7 \
-	libffi-3.0.9nb1 \
-	python27-2.7.2nb2 \
-	py27-setuptools-0.6c11nb1 \
-	py27-simplejson-2.1.1 \
-	erlang-14.1.4nb1 \
-	rabbitmq-2.7.1
+	perl-5.34.0nb3 \
```
Comment (danmcd) [inline Makefile:21]: Not a code nit, but a bug-report nit.  Would appreciate seeing in the ticket why versions are being picked IF AND ONLY IF they aren't just the "this is the latest/greatest/default in 21.4 pkgsrc."  `rabbitmq271-2.7.1` e.g.

Also make sure test notes are in ticket.

**Review by danmcd — COMMENTED** [review]
I'm okay with the code changes once I understand (and it's put-in-the-ticket sort of documentation) if any unusual package changes happened above-and-beyond, "Well, it's 2021.4 now."

File: Makefile (hunk 1 above)
Comment (bahamat) [reply Makefile:21]: In most cases these are the *only* versions in pkgsrc. For erlang and rabbitmq, the versions are the ones ported forward from the 2011 image.

Comment (danmcd) [reply Makefile:21]: Please state 👆 in the ticket prior to integration.

**Review by danmcd — APPROVED** [review]
Subject to updating the ticket.

**Pull request merged**